widgetastic_manageiq/__init__.py

```diff
@@ -1148,6 +1148,20 @@ def __init__(self, parent, text, logger=None):
         self.text = text
 
 
+class SSUINavDropdown(NavDropdown):
+    """The SSUI dropdowns used eg. in navigation. Usually located in the top navbar.
+       expand locator is different in SSUI from OPS UI"""
+
+    @property
+    def expandable(self):
+        try:
+            self.browser.element('./button/span[contains(@class, "caret")]', parent=self)
+        except NoSuchElementException:
+            return False
+        else:
```

Comment:
Consider omitting the ``else`` (just for readability as it's not needed here for this logic).